salt/output/json_out.py

```diff
@@ -54,7 +54,7 @@ def output(data):
     '''
     try:
         if 'output_indent' not in __opts__:
-            return json.dumps(data, default=repr, indent=4)
+            return json.dumps(data, default=repr, indent=4, encoding='latin1')
```

Comment:
Ok, what do you need in detail? I will not post the complete output as I must remove the most parts for DP reasons. 
Also I think the source issue comes from the pkg exec module in this case.